_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -4,6 +4,13 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### Templates DSL documentation
+
+Foreman now has documentation for DSL used for templates writing. This should
+make template writing process easier for non-Rubyists as well as for people who
+do know less about certain methods Foreman provides. Documentation is available
+under `https://<fqdn>/templates_doc`.
```

Comment:
"This documentation will be gradually enhanced in future releases to cover all available methods and macros."